ansible_collections/arista/avd/molecule/eos_designs_unit_tests_v4.0/intended/structured_configs/DC1-CL1A.yml

```diff
@@ -241,6 +241,16 @@ management_api_http:
   enable_https: true
   default_services: false
 vlans:
+- id: 4090
```

Comment:
These changes are OK, since eos_designs_unit_tests_v4.0 does not have any `convert_dict` functionality, so it breaks when we merge inside `yaml_templates_to_facts`.